### ISSUE TITLE
nanomath 1.0 - ave_qual commented out but tests still check for it

### ISSUE THREAD

**Issue by smoe** [wdecoster/nanomath#16]
Hi Wouter,

The subject line kind of summarises it already. You may be interested to know about the [Debian package we prepared for nanomath](https://tracker.debian.org/pkg/python-nanomath) as part of the bimonthly Debian Med Covid-19 sprint - I was just about to update the package when the autotests made this noise.

I have no idea if your reverse dependencies are using ave_qual. With version 1.0 you are of course completely free to change your API, but admittedly I wish you'd just formally deprecate it and remove it in, say, 1.1 or so.  Would you allow me to comment these lines back in? Or shall I remove the test?

Cheers,
Steffen 

```
I: pybuild base:217: cd /home/moeller/git/med-team/python-nanomath/.pybuild/cpython3_3.8_nanomath/build; python3.8 -m unittest discover -v 
test_ave_qual (nanomath.test_nanomath.NanomathTest)
Test average quality calculation. ... ERROR

======================================================================
ERROR: test_ave_qual (nanomath.test_nanomath.NanomathTest)
Test average quality calculation.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/moeller/git/med-team/python-nanomath/.pybuild/cpython3_3.8_nanomath/build/nanomath/test_nanomath.py", line 9, in test_ave_qual
    mq = nm.ave_qual(quals, qround=True)
AttributeError: module 'nanomath' has no attribute 'ave_qual'
```

**Comment by wdecoster** [comment]
Hi Steffen,

I commented it out when I realized that, as far as I know, this function isn't used anymore by any of the other packages. The function is duplicated in nanoget and NanoFilt. 

I thought I silenced that test as well: https://github.com/wdecoster/nanomath/blob/master/nanomath/test_nanomath.py
But seems that's not the case, or I misunderstood something of how unittest works.

It would probably be better if the test is performed there, I guess. In retrospection, splitting my code over (so many) multiple packages probably was a mistake, and I already deprecated one submodule a longer while ago when I moved the code to NanoPlot and NanoComp itself.

What would you suggest?

Cheers,
Wouter

**Comment by smoe** [comment]
Hi Wouter,

I think I would leave it in for now and flag it as described in
https://python-deprecated.readthedocs.io/en/latest/tutorial.html.

Your https://github.com/wdecoster/nanomath/blob/master/README.md is also mentioning ave_qual still and should possibly give references to nanoget and NanoFilt.

Best,
Steffen

**Comment by wdecoster** [comment]
I've fixed this in v1.0.1, sorry for the inconvenience!

**Comment by smoe** [comment]
Hi Wouter,
Thank you tons. I have just uploaded 1.0.1 to the distribution (as an update to 0.23.2). Please ping me if there is something else that you would like to see distributed with Debian.
Cheers,
Steffen
